pkg/collector/corechecks/oracle-dbm/activity.go

```diff
@@ -331,10 +331,13 @@ func (c *Check) SampleSession() error {
 	var sessionRows []OracleActivityRow
 	sessionSamples := []OracleActivityRowDB{}
 	var activityQuery string
+	var maxSqlTextLength int
 	if c.isRDS || c.isOracleCloud {
 		activityQuery = ACTIVITY_QUERY_DIRECT
+		maxSqlTextLength = 4000
```

Comment:
Is the `maxSqlTextLength` 4000/1000 defined by the cloud provider? Could we make them a global const?

Reply:
In the query for the self-hosted systems, we are selecting `V$SQLSTATS.SQL_FULLTEXT`. The field is `4000` characters wide, but because of the Oracle bug "Bug 27760729 : V$SQLSTAT.SQL_FULLTEXT DOES NOT SHOW COMPLETE SQL STMT". The query version for the cloud providers selects `V$SQL.SQL_FULLTEXT` which isn't affected by the bug. I'll define the constants and document it in the comments.